### PR TITLE
adding xui-webapp

### PR DESCRIPTION
**Pull request by fazanki**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

https://tools.hmcts.net/jira/browse/EUI-524

INFO: Found secret xui-webapp in environment sandbox
INFO: Found secret xui-webapp in environment demo
INFO: Found secret xui-webapp in environment aat
INFO: Found secret xui-webapp in environment prod
INFO: Found secret xui-webapp in environment perftest
INFO: Found secret xui-webapp in environment ithc

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -97,6 +98,11 @@ data "azurerm_key_vault_secret" "microservice_keys" {
   count     = "${length(local.microservice_key_names)}"
 }
 
+data "azurerm_key_vault_secret" "s2s_key" {
```
Comment (adusumillipraveen) [inline infrastructure/main.tf]: Why is this added?

Comment (fazanki) [reply infrastructure/main.tf]: I have no idea t by honest... it is mentioned in the read me the file so I thought to put it there 
do u want me to remove it?

Comment (adusumillipraveen) [reply infrastructure/main.tf]: yes, you need to remove it,  the document means you need to generate secrets manually  : https://github.com/hmcts/service-auth-provider-app#generating-secret and set them to vaults in all envs
https://github.com/hmcts/service-auth-provider-app#writing-a-secret-into-all-the-needed-vaults

Comment (fazanki) [reply infrastructure/main.tf]: ok we have generated the secrets manually - and I have removed the code

Comment (adusumillipraveen) [reply infrastructure/main.tf]: Can u pls address this as well ?
https://github.com/hmcts/service-auth-provider-app/pull/162#pullrequestreview-267882271

Comment (fazanki) [reply infrastructure/main.tf]: @adusumillipraveen that has been addressed now

**Comment by fazanki** [comment]
unfortunately, I don't have JDK 8 installed but 12 and because need 12 version for the other docker apps that i am running so I can't run the code locally. Tbh I am not java dev and trying to find my feet with this configuration... @adusumillipraveen

**Comment by timja** [comment]
You don't need to run the app.
